features/product/viewing_product_in_admin_panel/viewing_product_for_particular_channel.feature

```diff
@@ -0,0 +1,20 @@
+@viewing_products
+Feature: Viewing a product for a particular channel
+    In order to choosing view a product in each channel
```

Comment:
`In order to easily check product details as seen by a Customer`? cc @CoderMaggie